common-libs/tools-common/src/services/JwtService.ts

```diff
@@ -1,32 +1,35 @@
-import DidDocumentService from './DidDocumentService'
 import { decodeToken } from 'jsontokens'
 import encode from 'base64url'
 import { randomBytes } from '../shared/randomBytes'
 
-import { DEFAULT_JWT_EXPIRY_MS } from '../_defaultConfig'
+const JwtService = {
+  DEFAULT_JWT_EXPIRY_MS: 10 * 60 * 1000,
 
-export default class JwtService {
-  static getDidFromToken(jwt: string) {
+  keyIdToDid: (keyId: string) => {
```

Comment:
@inga-affinidi why this logic is moved there? This related logic how to get did from keyId its about didDocument and not JWT really

Reply:
Maybe I misunderstood things.
`JwtService` uses it (e.g. in `buildJWTInteractionToken`), and it is totally independent from the rest of affinidi code, so I thought it should belong here.

(Originally I tried to break `did-auth-lib -> common` dependency, replacing it with `did-auth-lib -> tools-common`, before I realized I should break `internal-api-clients -> did-auth-lib` instead)